File: backend/src/main/java/top/vuhe/entity/equipment/dto/StatusDTO.java
package top.vuhe.entity.equipment.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

/**
 * @author zhuhe
 */
@Data
@AllArgsConstructor
@RequiredArgsConstructor
public class StatusDTO {
    /**
     * 通道号
     */
    private Integer channel;
    /**
     * 状态号
     * <p>
     * 此号需要有状态值对应
     */
    private Integer status;
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/entity/equipment/dto/StatusDTO.kt
package top.vuhe.sw.entity.equipment.dto

typealias StatusDTO = List<StatusPair>

/**
 * 状态传送包
 *
 * @property channel 通道号
 * @property status  状态号
 */
data class StatusPair(
    val channel: Int?,
    val status: Int?
)
